src/sentry/notifications/utils/participants.py

```diff
@@ -468,10 +469,9 @@ def get_fallthrough_recipients(
     elif fallthrough_choice == FallthroughChoiceType.ACTIVE_MEMBERS:
         return user_service.get_many(
             filter={
-                "user_ids": project.member_set.order_by("-user__last_active").values_list(
-                    "user_id", flat=True
-                )
-            }
+                "user_ids": project.member_set.values_list("user_id", flat=True),
+                "order_by": UserOrderBy.LAST_ACTIVE_DESC,
```

Comment:
Two questions:
1.  Is ordering by last active valuable in reality?  We don't order by any other set of returned recipients, it honestly seems strange.
2.  What about doing sort in memory rather than continuing to increase complexity and contract of RPC?  We already have to ship a list of all users, so there's no a significant cost to doing a sort on the same list in python.

Reply:
3.  No test seems to actually validate that sorting occurs at all -- is it a real requirement?

Reply:
> Is ordering by last active valuable in reality? We don't order by any other set of returned recipients, it honestly seems strange.

The intent of the code is to notify the twenty most recently active users that are members on the project. From vague memory of this work it was to help increase the relevance of notifications, and notify users who aren't active.

> What about doing sort in memory rather than continuing to increase complexity and contract of RPC? We already have to ship a list of all users, so there's no a significant cost to doing a sort on the same list in python.

Yeah, we could do the sort in memory. I was concerned about having to sort the entire list only to slice off 20 records, but we're still fetching all the records and slicing off from what could be a large result set.

I can back out the additional parameter and do sorting & slicing in the region side.

> No test seems to actually validate that sorting occurs at all -- is it a real requirement?

If we're only going by the tests, sorting isn't covered, but the code looks pretty intentional.

Reply:
@markstory What if we denormalized user.last_active into `OrganizationMember` the way we do with `user_is_active` and `user_email`?  Then we can make the entire query locally and not bleed any logic into the region silo.

Reply:
> What if we denormalized user.last_active

That would add a lot of cross-region chatter. For active users, last_active updates every roughly every 5 minutes.

Reply:
That value is configurable and easily can be taken to say, a day, while still being useful for this logic.

A mid term goal is that notifications not depend on control silo, that's what I'm pondering here.

Reply:
In any case, I'll approve this PR